dev/io.openliberty.checkpoint_fat_mail/test-applications/IMAPTestingApp/resources/META-INF/permissions.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No need for `permissions.xml` file. Remove this.

Reply:
done